src/fiber/fiber.mli

```diff
@@ -298,6 +298,10 @@ module Stream : sig
     val sequential_iter : 'a t -> f:('a -> unit fiber) -> unit fiber
 
     val parallel_iter : 'a t -> f:('a -> unit fiber) -> unit fiber
+
+    val append : 'a t -> 'a t -> 'a t
+
+    val cons : 'a t -> 'a -> 'a t
```

Comment:
Having `cons t x` produces a stream that starts with `x` and is followed by elements of `t` is counter-intuitive. On this occasion, I think it's fine to break the rule that `t` comes first. Or the function should be renamed.

Reply:
I flipped it